### PR TITLE
Sets up the hamilton ui to run on sqlite + be installed/run with a pip install command

### PR DESCRIPTION
**Pull request by elijahbenizzy**
You can run it with:
- `python manage.py runserver --settings=server.settings_mini 0.0.0.0:8241` from `ui/backend/server`
- `npm run start` from `ui/frontend/server`
## Checklist

- [x] PR has an informative and human-readable title (this will be pulled into the release notes)
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code passed the pre-commit check & code is left cleaner/nicer than when first encountered.
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future TODOs are captured in comments
- [ ] Project documentation has been updated if adding/changing functionality.

### CODE REVIEW

**Comment by elijahbenizzy** [comment]
This is working -- you can:
```
pip install "sf-hamilton[serve]==1.65.0rc0"
hamilton-serve
```

**Comment by elijahbenizzy** [comment]
TODO:
- [ ] Docs -- add to the UI section
- [ ] README -- add getting started to the front page
- [ ] Test out the docker iamges so they still world
- [ ] Add in example projects

File: MANIFEST.in [hunk 1]
```diff
@@ -1,4 +1,6 @@
 include requirements.txt
 include requirements-test.txt
+include hamilton/server/requirements-mini.txt
+recursive-include hamilton/server/build *
```
Comment (skrawcz) [inline MANIFEST.in:4]: I'm confused, does this mean every Hamilton package will include the contents of build? That doesn't sounds right to me.

File: setup.py [hunk 2]
```diff
@@ -44,7 +52,14 @@ def load_requirements():
...
     url="https://github.com/dagworks-inc/hamilton",
-    packages=find_packages(exclude=["tests"]),
+    packages=find_packages(exclude=["tests"], include=["hamilton", "hamilton.*"]),
+    # package_data={
+    #     "hamilton": [
+    #         "ui/server/build/**/*",
+    #         "hamilton/server/**/*",
+    #         "hamilton/server/build/**/*"
+    #     ]
+    # },
```
Comment (skrawcz) [inline setup.py:62]: remove.

File: MANIFEST.in (hunk 1 above)
Comment (elijahbenizzy) [reply MANIFEST.in:4]: Yeah it's not easy to disambiguate, although we could probably hack it in

File: ui/backend/setup.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        "ui": load_requirements(),
+    },
+    entry_points={
+        "console_scripts": [
+            "h_experiments = hamilton.plugins.h_experiments.__main__:main",
+            "hamilton = hamilton.cli.__main__:cli",
+            "hamilton-serve = hamilton.server.__main__:run",
+            "hamilton-admin-build-ui = hamilton.admin:build_ui",
+            "hamilton-admin-build-and-publish = hamilton.admin:build_and_publish",
+        ]
```
Comment (skrawcz) [inline ui/backend/setup.py:90]: are these correct?

File: ui/backend/setup.py [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        ],
+        "experiments": [
+            "fastapi",
+            "fastui",
+            "uvicorn",
+        ],
+        "diskcache": ["diskcache"],
+        "cli": ["typer"],
+        "sdk": ["sf-hamilton-sdk"],
+        "ui": load_requirements(),
```
Comment (skrawcz) [inline ui/backend/setup.py:81]: these aren't needed here, right?

**Review by skrawcz — APPROVED** [review]
Just need to clean up the UI setup.py I think.

**Pull request merged**